client/packages/android/app/build.gradle

```diff
@@ -18,14 +18,20 @@ android {
     }
     signingConfigs {
         release {
-            def Properties localProps = new Properties()
-            localProps.load(new FileInputStream(file('../local.properties')))
+            // Check for release and if in release mode try to load signing info
+            if (project.gradle.startParameter.taskNames.any { it.toLowerCase().contains('release') }) {
```

Comment:
gradle files are evaluated fully unless we do this conditional, previously had errors when trying to do debug build.